specs/guard-jwt.spec/src/main/scripts/io/aklivity/zilla/specs/guard/jwt/config/guard-keys-dynamic.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "test",
+    "guards":
+    {
+        "jwt0":
+        {
+            "type": "jwt",
+            "options":
+            {
+                "issuer": "https://aklivity.us.auth0.com/",
+                "audience": "http://localhost:8080/tasks",
+                "keys": "https://aklivity.us.auth0.com/.well-known/jwks.json",
```

Comment:
Perhaps use a different URL from the default when specified explicitly, otherwise we cannot tell that it is being used instead of the default.

Reply:
ok, i reverted this change to the original one